dev/com.ibm.ws.webcontainer/src/com/ibm/wsspi/webcontainer/WebContainerRequestState.java

```diff
@@ -246,17 +246,24 @@ public void setCookieAttributes(String cookieName, String cookieAttributes) {
             }
         }
 
+        System.out.println("Num Attr values: " + attribute.length) ;
```

Comment:
Take out or change to `Tr.debug()` statement.

Reply:
Fixed